infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -126,7 +126,7 @@ module "psoxy-google-workspace-connector" {
   secret_bindings = merge({
     # as SERVICE_ACCOUNT_KEY rotated by Terraform, reasonable to bind as env variable
     SERVICE_ACCOUNT_KEY = {
-      secret_id      = module.google-workspace-key-secrets[each.key].secret_ids["PSOXY_${replace(upper(each.key), "-", "_")}_SERVICE_ACCOUNT_KEY"]
+      secret_id      = module.google-workspace-key-secrets[each.key].secret_secret_ids["PSOXY_${replace(upper(each.key), "-", "_")}_SERVICE_ACCOUNT_KEY"]
```

Comment:
secret_secret_ids is kind of confusing

Reply:
yeah, agreed; unfortunately was introduced in earlier version so a breaking change to just rename, but I'll mark deprecated, duplicate it with a better name, and use that.